.github/workflows/release-arm.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
           curl https://luet.io/install.sh | sudo sh
           IMG=$(cat build/IMAGE).img
           sudo tar cvf build.tar build
-          sudo luet util pack $IMG build.tar image.tar
+          sudo luet util pack quay.io/kairos/kairos-${{ matrix.flavor }}-img:${{ matrix.k3s_version }} build.tar image.tar
```

Comment:
I dont get it, shouldnt the build/IMAGE file have the proper version? Seems to properly pack and load, only fail on docker push because we are not using the same IMG variable no?

Reply:
I think the Robles with the release ones is that I didn't tag them properly so that's why they don't match. And I might have to also fix it for the kairos repo, where is green but I think it points to the wrong one 😞

Reply:
yeah that's why I want to get rid now of IMG because inside `build/IMAGE` there will be a `kairos-FLAVOR:K3S_VERSION` which actually needs to be `kairos-FLAVOR-img:K3S_VERSION`

Reply:
ok makes sense, closing this one

Reply:
agreed, my point here is to try to fix the issue with a small change here to avoid blocking the release - we can come back at this later anytime to address it completely, but for the moment, I'd suggest to workaround in order to make the CI pass for this one